chromadb/api/segment.py

```diff
@@ -832,15 +848,13 @@ def _validate_dimension(
 
     @trace_method("SegmentAPI._get_collection", OpenTelemetryGranularity.ALL)
     def _get_collection(self, collection_id: UUID) -> t.Collection:
-        """Read-through cache for collection data"""
-        if collection_id not in self._collection_cache:
-            collections = self._sysdb.get_collections(id=collection_id)
-            if not collections:
-                raise InvalidCollectionException(
-                    f"Collection {collection_id} does not exist."
-                )
-            self._collection_cache[collection_id] = collections[0]
-        return self._collection_cache[collection_id]
+        """Get a collection database."""
```

Comment:
hmmm, i think removing this cache in single node is worth a thought?

Reply:
That should be fine. Essentially, Sqlite will cache?